python/lsst/ctrl/mpexec/separablePipelineExecutor.py

```diff
@@ -84,9 +84,15 @@ class SeparablePipelineExecutor:
     butler : `lsst.daf.butler.Butler`
         A Butler whose ``collections`` and ``run`` attributes contain the input
         and output collections to use for processing.
+    extend_run : `bool`, optional
+        True if extending an existing run instead of creating a new empty run.
+        Quanta with complete outputs will be skipped. Partial outputs from
+        failed quanta will be removed if ``clobberOutputs`` is `True`,
+        otherwise an exception will be raised.
     clobber_output : `bool`, optional
-        If set, the pipeline execution overwrites existing output files.
-        Otherwise, any conflict between existing and new outputs is an error.
+        If set, the pipeline execution overwrites existing output files from
+        previously failed quanta. Otherwise, any conflict between existing and
+        new outputs is an error.
```

Comment:
If one wanted to overwrite all outputs unconditionally, how would one do that now?

Reply:
In `pipetask` land we have a couple of options (--replace-run and --prune-replaced) to clean up run collection, but it only works on the whole collections, not individual quanta. It should be possible to write few lines of code to explicitly remove quanta outputs from Butler.